### PR TITLE
cropgui: update to 0.4.

### PR DESCRIPTION
**Pull request by zdykstra**

### CODE REVIEW

File: srcpkgs/cropgui/template [hunk 1]
```diff
@@ -1,17 +1,17 @@
 # Template file for 'cropgui'
 pkgname=cropgui
-version=0.3
+version=0.4
 revision=1
 hostmakedepends="python"
 makedepends="python"
```
Comment (Piraty) [inline srcpkgs/cropgui/template]: please change to py3 if possible, upstream's history indicates it support is already

**Comment by pullmoll** [comment]
See efb347e69f3f4cef953870405844b53dbefbc615
I could not get it build right with `python3` and w/o `pygtk`.

**Comment by zdykstra** [comment]
Thanks - I haven't had the time to address this.

**Comment by pullmoll** [comment]
It doesn't work for me though :-/
It looks like `cropgui_common.py` is not compiled or not found. I'm not the python type and would ask others to take a look at the template and add whats missing or fix whats wrong.

**Comment by zdykstra** [comment]
> It doesn't work for me though :-/
> It looks like `cropgui_common.py` is not compiled or not found. I'm not the python type and would ask others to take a look at the template and add whats missing or fix whats wrong.

Ahh, take a look at the shebang in /usr/bin/cropgui - ` #!/usr/bin/python3` 
cropgui_common.py is being installed to `/usr/lib/python2.7/site-packages/cropgui_common.py`.  

I've tried just flipping the shebang to python2.7, but other things are then broken. I unfortunately don't have time to dig any deeper right now.

**Pull request closed without merging**